specs/SUMMARY.md

```diff
@@ -59,4 +61,5 @@
     - [SuperchainWETH](./interop/superchain-weth.md)
   - [Security Council Safe](./experimental/security-council-safe.md)
   - [OP Stack Manager](./experimental/op-stack-manager.md)
+  - [Governance Token](./experimental/gov-token.md)
```

Comment:
ik we will eventually just have on gov token doc, but maybe we should differentiate the name here, like
```suggestion
  - [Governance Token - Advanced Delegation](./experimental/gov-token.md)
```